lib/components/ContactUsForm.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  HStack,
+  Input,
+  Center,
+  Box,
+  Button,
+  Textarea,
+} from '@chakra-ui/react';
+
+interface contactFormProps {}
+
+interface NameFormProps {
+  firstNameInput: string;
+  lastNameInput: string;
+  setFirstInput: any;
```

Comment:
🔧 any is Not Allowed in this household!

Instead either use the built in set state type from React, or a more readable function type like:
`(string) => void`

Reply:
ah I forgot to go back and change it, yep thank you!